src/orders/types.ts

```diff
@@ -46,6 +46,7 @@ type OrderByDirection = "asc" | "desc";
 
 export type OrderAPIOptions = {
   protocol?: OrderProtocol;
+  protocolAddress?: string;
```

Comment:
The key in the JSON has an underscore: `protocol_address`

Reply:
thank you, added in https://github.com/ProjectOpenSea/opensea-js/pull/846/commits/c523963ade8835604528606da06bd2d2e3ac2cdf